libs/rest/rest-server-impl/src/main/kotlin/net/corda/rest/server/impl/context/ClientHttpRequestContext.kt

```diff
@@ -58,6 +59,10 @@ internal class ClientHttpRequestContext(private val ctx: Context) : ClientReques
         addHeaderValues(authMethods)
     }
 
+    override fun addPasswordExpiryHeader(expiryStatus: PasswordExpiryStatus) {
+        ctx.res.addHeader(Header.WARNING, "PasswordExpiryStatus is $expiryStatus")
```

Comment:
```suggestion
        ctx.res.addHeader(Header.WARNING, "199 - PasswordExpiryStatus is $expiryStatus")
```
Just to be able to comply with: https://www.rfc-editor.org/rfc/rfc7234#section-5.5.5